src/components/browse/datasetlist/datasetheader/datasetdetails/datasetFiles.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Row } from "react-bootstrap";
+import DatasetDetailsLayout from "./datasetdetailslayout/datasetDetailsLayout";
+import { FileEarmark } from "react-bootstrap-icons";
+
+interface dataSetFilesProps {
+  filesList: string[];
+}
+
+const DatasetFiles = (props: dataSetFilesProps) => {
+  return (
+    <DatasetDetailsLayout
+      icon={<FileEarmark size={32} />}
```

Comment:
Lets go with the grey from the wireframes. I checked with Koray and he would like to stick with the design.

Reply:
Updated.